crates/modelardb_server/tests/integration_test.rs

```diff
@@ -763,6 +806,32 @@ fn test_optimized_query_results_equals_non_optimized_query_results() {
     assert_eq!(optimized_query, non_optimized_query);
 }
 
+#[test]
+fn test_can_update_uncompressed_reserved_memory_in_bytes() {
```

Comment:
Maybe refactor the code in these two tests into a shared function as the two tests only seem to differ by a string and an index.

Reply:
Fixed.